skimage/filters/thresholding.py

```diff
@@ -366,3 +367,97 @@ def threshold_li(image):
             new_thresh = temp + tolerance
 
     return threshold + immin
+
+def threshold_minimum(image, nbins=256, bias='min') :
+    """ Return threshold value based on minimum method
+
+    Parameters
+    ----------
+    image : array
+        Input image.
+    nbins : int, optional
+        Number of bins used to calculate histogram. This value is ignored for
+        integer arrays.
+    bias : string, optional
+        'min', 'mid', 'max' return lowest, middle, or highest pixel value
+        with minimum histogram value
+
+    Returns
+    -------
+    threshold : float
+        Upper threshold value. All pixels intensities that less or equal of
+        this value assumed as foreground.
+        May return 0 if the algorithm fails
+
+    References
+    ----------
+    Prewitt, JMS & Mendelsohn, ML (1966), "The analysis of cell images",
+    Annals of the New York Academy of Sciences 128: 1035-1053
+
+    Examples
+    --------
+    >>> from skimage.data import camera
+    >>> image = camera()
+    >>> thresh = threshold_minimum(image)
+    >>> binary = image > thresh
+    """
+
+    # Smooth histogram with a 3-element kernel: [1, 1, 1] / 3
+    def smooth_histogram(hist):
```

Comment:
`scipy.ndimage.filters.uniform_filter` should be a good replacement for this function.